shared/src/business/entities/DocketEntry.ts

```diff
@@ -189,7 +188,11 @@ export class DocketEntry extends JoiValidationEntity {
   public signedJudgeUserId?: string;
   public strickenBy?: string;
   public strickenByUserId?: string;
-  public workItem?: any;
+
+  // These are optional fields set solely for the UI in certain cases.
+  public qcComplete?: boolean;
+  public qcViewed?: boolean;
+  public workItemId?: string;
```

Comment:
This represents the minimum amount of data the UI needs associated with a docket entry.

I wanted to make an entirely new type `UIDocketEntry` that was `RawDocketEntry & {these fields}` to avoid further cluttering `DocketEntry`, but `genericHandler`, which reconstructs entities, made this difficult: without the fields here on the entity, they were wiped away when sending data back to the client (since `genericHandler` said, "Oh look, a `Case`, let me reconstruct it from scratch--oops, dropped those custom fields of yours!"). With the fields thus needing to be here in `DocketEntry` anyway, it felt useless to have the new type.